.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "deps/azure-macro-utils-c"]
```

Comment:
are any of these other submodules needed anymore?

Reply:
look to remove them?

Reply:
I had removed that, I don't know why it is here. I'll remove again. 

---
In reply to: [597935610](https://github.com/Azure/azure-ulib-c/pull/62#discussion_r597935610) [](ancestors = 597935610)